### PR TITLE
Add short pre-install hardware checklist

### PR DESCRIPTION
**Pull request by redshiftzero**
## Status

Ready for review

## Description of Changes

Changes proposed in this pull request:
 * Adds reusable text snippet for a very short pre-install checklist. The goal of this checklist is to provide the "tl;dr" for people that do not want to read the long hardware document under "Install SecureDrop"
 * Adds "checklists" section to sidebar:

screen shot 2017-08-16 at 6 09 27 pm
  * Adds the hardware checklist to the checklists section and to the top of the long hardware document under "Install SecureDrop"

This complements the pre-onsite checklist that @justintroutman is working on. 

## Testing

* Build docs locally and make sure everything renders well
* See if there's anything _critical_ I've left off the pre-install checklist. Note that the checklist is intentionally terse and only what is _necessary_ for a SecureDrop install to be successful should be on it. 

## Deployment

None, documentation only. 

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/includes/pre-install-hardware.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+* Two computers with memory and hard drives to use as the SecureDrop servers.
```
Comment (olivemartini) [inline docs/includes/pre-install-hardware.txt]: For checklist language, replace every spelled number with a numeral, i.e. "one"--->"1"

File: docs/includes/pre-install-hardware.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+* Mouse, keyboard, monitor (and necessary dongle or adapter) for
+  installing the servers.
+* Dedicated physical computers for the Admin, Journalist, and Secure Viewing
+  Station that can boot to Tails. At *minimum* this is two more computers.
```
Comment (olivemartini) [inline docs/includes/pre-install-hardware.txt]: Remove "more." Should read: "At *minimum* this is 2 computers."

**Comment by redshiftzero** [comment]
Changes made and squashed into existing commits, take a look and let me know if that looks good 😸

**Comment by redshiftzero** [comment]
Great, thanks @olivemartini!

**Pull request merged**